docs/api/cypress-api/ensure.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: Cypress.ensure
+---
+
+`Cypress.ensure.method()` is a collection of helper methods for verifying
+conditions while executing Cypress commands. They are mostly useful when writing
+[custom queries](/api/cypress-api/custom-queries) or
+[commands](/api/cypress-api/custom-commands).
```

Comment:
```suggestion
[custom commands](/api/cypress-api/custom-commands).
```

Reply:
or remove custom from the query link if you'd prefer `custom [queries](link) or [commands](link)`